ts/att/timeSpan.ts

```diff
@@ -0,0 +1,34 @@
+export class TimeSpan {
+
+    public totalSeconds: number;
```

Comment:
@DheenodaraRao  this should be `readonly` , means it cannot be changed after initialization (by calling constructor).
So, it should be 
```ts
public readonly totalSeconds: number;
```

Moreover, please do add `totalMinutes` and `totalHours` as well.

Reply:
Done